peachjam/admin.py

```diff
@@ -167,7 +167,11 @@ def attachment_link(self, obj):
 class SourceFileInline(BaseAttachmentFileInline):
     model = SourceFile
     form = SourceFileForm
-    readonly_fields = (*BaseAttachmentFileInline.readonly_fields, "source_url")
+    readonly_fields = (
```

Comment:
Do you have a screenshot of what this looks like? I'm wondering if it's just too much detail and we can simply not have it at all.

Reply:
It may be too much detail. We can exclude the field.